.circleci/config.yml

```diff
@@ -107,9 +119,12 @@ workflows:
           requires:
             - test-node
             - test-py
-      - deploy:
+      - backup:
```

Comment:
I think backup should run only on master

Reply:
Yes, I remove it for testing, add it back after passing the test.